src/coreclr/vm/methodtable.inl

```diff
@@ -1157,6 +1157,32 @@ inline PTR_BYTE MethodTable::GetNonGCThreadStaticsBasePointer()
     return pTLM->GetNonGCStaticsBasePointer(this);
 }
 
+//==========================================================================================
+inline PTR_BYTE MethodTable::GetGCThreadStaticsBaseHandle()
+{
+    CONTRACTL
+    {
+        NOTHROW;
+        GC_NOTRIGGER;
+        MODE_COOPERATIVE;
+    }
+    CONTRACTL_END;
+
+    // Get the current thread
+    PTR_Thread pThread = dac_cast<PTR_Thread>(GetThread());
+
+    // Get the current module's ModuleIndex
+    ModuleIndex index = GetModuleForStatics()->GetModuleIndex();
+
+    PTR_ThreadLocalBlock pTLB = ThreadStatics::GetCurrentTLB(pThread);
+
+    PTR_ThreadLocalModule pTLM = pTLB->GetTLMIfExists(index);
+    if (pTLM == NULL)
+        return NULL;
+
+    return dac_cast<PTR_BYTE>(pTLM->GetPrecomputedGCStaticsBaseHandle());
```

Comment:
This is almost a copy of the function above, but that one doesn't use `dac_cast<PTR_BYTE>`; is that an omission?

Reply:
which method are you referring?